activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+*   Allow Range#=== and Range#cover? on Range
+
+    `Range#cover?` can now accept a range argument like `Range#include` and
+    `Range===`. `Range#===` works correctly on Ruby 2.6. `Range#include?` is moved
+    into a new file, with these two methods.
+
+    To use: `require "active_support/core_ext/compare_range"`.
+
+    *Requiring core_ext/range/include_range now causes a deprecation warnning*
```

Comment:
"warnning" -> "warning"

Can you also add `active_support/` to the start of the filename, to match the example above?

Reply:
We're telling the user that requiring this file is deprecated, but not what they need to do instead - it's pretty easy to _infer_ that they should now require `active_support/core_ext/compare_range`, but ideally we'd explicitly state that.

Reply:
Right on.
Sorry about all the typos.